src/middlewares/schema.spec.ts

```diff
@@ -183,12 +183,45 @@ describe('on each field', () => {
     const { context } = await chain.run({ params: { foo: '' } });
     expect(context.errors).toHaveLength(1);
   });
+
+  it('use default config when specified', async () => {
+    const chain = checkSchema({
+      undef: {
+        default: {
+          options: 10,
+        },
+      },
+    })[0];
+
+    const req = { query: { undef: undefined } };
+    const { context } = await chain.run(req);
+
+    expect(context.getData().every(instance => instance.value === 10)).toBe(true);
+    expect(req.query.undef).toBe(10);
```

Comment:
Can we check that also `req.body.undef`, ... have value of `10`. We will fix this testing problem with fixtures soon :)